chart/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for loki-rule-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: eu.gcr.io/opsgy-com/loki-rule-operator
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "loki-rule-operator"
+
+podAnnotations: {}
+
+podSecurityContext:
+  fsGroup: 2000
+  runAsUser: 10000
+
+securityContext:
+  readOnlyRootFilesystem: true
+
+service:
+  type: ClusterIP
+  port: 443
+
+resources:
+  limits:
+    memory: 50Mi
+    cpu: 50m
+  requests:
+    memory: 50Mi
+    cpu: 50m
+
+lokiConfigmapRulesNamespace: monitoring
```

Comment:
Maybe rename to 
```yaml
loki:
  rulesConfigMap:
    name: loki-rules
    namespace: default
```

Reply:
@steven166 I think the default to monitoring namespace _make sense_. This is where prometheus, loki, grafana, etc. is usually installed.
If you insist, I can change it to default but I think most people will change it to monitoring.

Reply:
I think it depends :) 
But maybe a good default would be the same namespace as where the operator is deployed into. 
So something like this: `{{ .Values.loki.rulesConfigMap.namespace | default .Release.Namespace }}`

Reply:
@steven166 Good point, will do.